README.md

```diff
@@ -113,13 +113,17 @@ export PYTHONPATH=${PWD}
 uvicorn --env-file=.env.development --app-dir=ahd2fhir main:app --reload --log-level=debug
 ```
 
+> **Note**
+> To enable reading FHIR DocumentReferences from a Kafka topic during development,
+> make sure to set the env var `KAFKA_ENABLED=true`
+
 Uses the environment configuration from the `.env.development` file. You will need to modify the `AHD_` env vars for
 your local deployment.
 
 ### Build and run using a locally-build image
 
 Note the use of `host.docker.internal` so the running container can still access the version of AHD launched via
```

Comment:
Impliziert die Nutzung von z.B. `host.docker.internal` im compose file nicht die explizite Nutzung von Docker? Ich frage, weil die Umbenennung von `docker-compose.yml` in `compose.yml` ja eine generische Verwendbarkeit des files suggeriert.

Reply:
stimmt schon, aber host.docker.internal wird zB. auch in podman wegen rückwärtskompatibilität unterstützt: https://github.com/containers/podman/issues/19361. Der korrekteste name wäre wohl `host.containers.internal` - bin mir aber nicht sicher ob das wiederrum docker unterstützt.